packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -272,6 +272,29 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('setZoomLevel() is not implemented.');
   }
 
+  /// Gets a list of video stabilization modes that are supported for the selected camera.
+  ///
+  /// Will return the list of supported video stabilization modes
+  /// on Android (when using camera_android_camerax package) and
+  /// on iOS. Will return an empty list on all other platforms.
+  Future<Iterable<VideoStabilizationMode>> getVideoStabilizationSupportedModes(
+      int cameraId) async {
+    throw UnimplementedError();
```

Comment:
nit: To match the other methods

```suggestion
    throw UnimplementedError('getVideoStabilizationSupportedModes() is not implemented.');
```